### DISCUSSION TITLE
Feed filter doesn't work

### DISCUSSION THREAD

**Discussion by ris58h** [community/community#28987] [Other Feature Feedback, Questions, & Ideas]
I've tried to apply a feed filter but it doesn't do anything.
I use Firefox for android. 
![Screenshot_20220810-161408_Firefox](https://user-images.githubusercontent.com/511242/183864220-0e63c322-8ac9-4b2d-9e7f-8b0fff4b8fb6.jpg)

**Comment by jfullstackdev** [comment]
can we see the feed in each filter ?

**Reply by ris58h** [reply]
There is no difference. When I click on any filter-checkbox it doesn't change anything.
